.github/actions/system-info/system-info.py

```diff
@@ -0,0 +1,32 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+import platform
+import sys
+
+print(f"{sys.platform=}", file=sys.stderr)
+
+if sys.platform == "linux":
+    platform_info = platform.freedesktop_os_release()
+    release = platform_info["VERSION_ID"]
+    os = platform_info["ID"]
+elif sys.platform == "darwin":
+    platform_info = platform.mac_ver()
+    release = platform_info[0]
+    os = "macos"
+elif sys.platform == "win32":
+    platform_info = platform.win32_ver()
+    release = platform_info[0]
+    os = "windows"
+else:
+    raise ValueError(f"Unsupported platform {sys.platform}")
+
+print(f"{platform_info=}", file=sys.stderr)
+print(f"{os=} {release=}", file=sys.stderr)
+
+id = f"{os}-{release}"
```

Comment:
This gives consistent output for windows and macos, but not for linux?
- `windows-<release>` 
- `macos-<release>`
- `<distro>-<version>`

Reply:
That's exected, you only have one flavor of `windows` & `macos` but multiple flavor for `linux`